File: Models/TrackData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Tracking_Task.Models
{
    public class TrackData
    {
        public int Id { get; set; }
        public int BooksId { get; set; }
        public Books Books { get; set; }
        //public int InivitedTableId { get; set; }
        //public InivitedTable InivitedTables { get; set; }
        public string Change { get; set; }
        public DateTime Trackingdate { get; set; }
    }
}

File: Tracking_task_angular/src/app/track-data.ts
import { Books } from './books';
export class TrackData {
    Id:number;
    booksId:number;
    books:any;
    change:string;
    trackingdate:any;
    constructor(){
        this.Id=0
        this.booksId=0;
        this.books=null;
        this.change="";
        this.trackingdate=null;
    }

}
